cartridges/int_adyen_SFRA/cartridge/scripts/hooks/payment/processor/adyen_credit.js

```diff
@@ -83,8 +86,8 @@ function Authorize(orderNumber, paymentInstrument, paymentProcessor) {
 
     if (result.IssuerUrl != '') {
         Transaction.commit();
-        session.custom.order = order;
-        session.custom.paymentInstrument = paymentInstrument;
+        session.custom.orderNo = order.orderNo;
+        session.custom.paymentMethod = paymentInstrument.paymentMethod;
```

Comment:
is paymentMethod unique ?

Reply:
No, it is not unique, but with the paymentMethod, you can retrieve the paymentInstrument of the order

Reply:
Can we then not better use a paymentInstrumentId that is unique so we can store this and retrieve it ?